src/app/components/SearchBar/SearchBar.tsx

```diff
@@ -20,7 +20,13 @@ const Z_INDEX_OFFSET = 2 // Set a custom offset
 const StyledSearchBarToggle = styled(SearchBarToggle)`
```

Comment:
You should be  able to add the type information here so you don't have to redefine it below.
```suggestion
const StyledSearchBarToggle = styled(SearchBarToggle)<{ searchIsVisible: boolean }>`
```